en/ch02.md

```diff
@@ -50,6 +50,34 @@ It has a countdown that allows you to change some settings before your wallet co
 ## Create new wallet from hardware
 On this page you can create a new wallet file from your hardware wallet device. This wallet file will be stored in your computer and every time you open it you will be asked to connect and unlock your hardware wallet. Monero GUI currently supports the hardware wallet models Ledger Nano S/X and Trezor Model T.
 
+### How hardware wallets works with Monero GUI
+
++ The main function of your hardware wallet is to store your Bitcoin seed in a device that never connects to the internet.
++ The Bitcoin seed stored in your device will be used to generate a Monero seed, which will not be displayed to you.
++ The Monero seed inside your device will be used to create two Monero private keys: a view key and a spend key.
++ Your private view key is exported from your hardware wallet device to the Monero GUI wallet.
++ Your private spend key (required to send transactions) never leaves your hardware wallet device.
+
+### Prepare your device
+
+If you are using a Ledger Nano S/X, please make sure:
```

Comment:
Perhaps add that the Ledger Live desktop / manager has to be closed?

Reply:
This is mentioned below, in "Prepare your computer" section